src/vip/batch_address_test_tool/processor.clj

```diff
@@ -63,7 +63,10 @@
 
 (defn retrieve-polling-locations*
   [ctx]
-  ctx)
+  (let [addresses (:addresses ctx)
+        polling-locations (doall (map #(civic-info/address->polling-location (:address %)) addresses))
```

Comment:
The `doall` is probably not needed here because the next line will eagerly realize all of the `polling-locations`. So that's one thing I think you can eliminate from this line.